makefile

```diff
@@ -135,8 +135,11 @@ es6: install
 ui-compile:
 	@cd ui && yarn -s compile
 
+ui-install:
+	@cd ui && yarn -s install
+
 ui-watch:
 	@cd ui && yarn -s watch
 
-ui-install:
-	@cd ui && yarn -s install
+ui-watch-nashorn:
+	@cd ui && yarn -s watch:nashorn
```

Comment:
I am not even sure we should provide multiple ways.

Reply:
I think in the long term, it would be good to get rid of this. In the short term, until we have confidence that Node and Nashorn provide functionally equivalent runtime environments, it will be useful to have the ability to debug in both environments. 

I can imagine a situation where a developer builds a component that renders beautifully on Node, but breaks when it is deployed to CODE. It would be useful at that stage for the developer to have access to a production-like debugging environment.